.github/ISSUE_TEMPLATE/Bug_report.yml

```diff
@@ -0,0 +1,33 @@
+name: Bug Report
+description: Create a report to help us improve SciPy
+title: "[Bug]: "
```

Comment:
NIT: SciPy generally follows the [convention](http://scipy.github.io/devdocs/dev/contributor/development_workflow.html#writing-the-commit-message) without brackets in the commits and something similar for Issue/PRs maybe like "BUG: ".

I personally don't feel one is better or worse. Maintainers can probably comment here and totally ignore my comment. Very very small thing. :))

Reply:
Good point! And actually linking to this section of the doc could be helpful. In general adding more links could be good here since we can have a nicer formatting.

Reply:
Thanks for pointing this out!